modules/core/include/opencv2/core/cv_cpu_dispatch.h

```diff
@@ -152,6 +152,11 @@
 #  define CV_VSX3 1
 #endif
 
+#if defined(EMSCRIPTEN)
+#  define CV_WASM_SIMD 1
```

Comment:
Usually this macro is used in OpenCV for check:
```
defined(__EMSCRIPTEN__)
```
Is there any difference?

How SIMD feature can be disabled (via CMake/.py script parameters)? (it is useful for debugging purposes)

Reply:
Thanks for review!
According to [Detecting Emscripten in preprocessor](https://kripken.github.io/emscripten-site/docs/compiling/Building-Projects.html#detecting-emscripten-in-preprocessor), the correct define to use is `__EMSCRIPTEN__`.
https://github.com/kripken/emscripten/pull/4665  introduced a strict build mode and removed the `EMSCRIPTEN` define. Therefore it is not recommended to use `EMSCRIPTEN` even though it still works in non-strict build mode.
I'll fix that then.

Reply:
For how to disable SIMD feature, it's decided with a .py script flag `--simd`. If you build with this flag, `CV_ENABLE_INTRINSICS` will be turned on, and then SIMD feature will be detected. And if not, only scalar version will be built.